sport/app/football/controllers/LeagueTableController.scala

```diff
@@ -157,28 +157,41 @@ class LeagueTableController(
 
           val smallTableGroup =
             table.copy(groups = table.groups.map { group => group.copy(entries = group.entries.take(10)) }).groups(0)
-          val htmlResponse = (atom: Option[InteractiveAtom]) =>
-            () =>
-              football.views.html.tablesList
-                .tablesPage(
-                  TablesPage(
-                    page,
-                    Seq(table),
-                    table.competition.url,
-                    filters(tableOrder),
-                    Some(table.competition),
-                    atom,
-                  ),
-                )
-          val jsonResponse = () =>
-            football.views.html.tablesList.tablesComponent(
-              table.competition,
-              smallTableGroup,
-              table.competition.fullName,
-              multiGroup = table.multiGroup,
-            )
-
-          futureAtom.map(maybeAtom => renderFormat(htmlResponse(maybeAtom), jsonResponse, page))
+
+          futureAtom.map { atom =>
```

Comment:
It doesn't look like the Euro 2024 tables are available any more, so could we just delete this atom code instead?

Reply:
Yes we can, I just didn't know if this atom is a pattern that might regularly need to be added or if it's something that was just added for euro 2024, but looking at this PR https://github.com/guardian/frontend/pull/27220 it seems it was added just for euro 2024. 

It makes sense to delete the atom if it's not needed any more. I'm just thinking if I should delete the part related to the league table as part of this PR and maybe create an issue to remove it from everywhere else as well?

Reply:
Ticket created for this: https://github.com/guardian/dotcom-rendering/issues/13783